src/chapters/ldap-auth.xml

```diff
@@ -369,6 +369,15 @@ dn: cn={4}guacConfigGroup,cn=schema,cn=config
                           here.</para>
                     </listitem>
                 </varlistentry>
+                <varlistentry>
+                    <term><property>ldap-member-attribute-type</property></term>
+                    <listitem>
+                          <para>Specify whether the attribute defined in 
```

Comment:
It feels like this is written in "short-hand" rather than following the style of the rest of the documentation.  Maybe a little more verbosity ;-).